test/Datadog.Trace.ClrProfiler.IntegrationTests/Helpers/SpanTestHelpers.cs

```diff
@@ -6,11 +6,12 @@
 
 namespace Datadog.Trace.ClrProfiler.IntegrationTests
 {
-    public class WebServerTestHelpers
+    public class SpanTestHelpers
     {
-        public static void AssertExpectationsMet(
-            List<WebServerSpanExpectation> expectations,
+        public static void AssertExpectationsMet<T>(
+            List<T> expectations,
             List<MockTracerAgent.Span> spans)
+            where T : SpanExpectation
```

Comment:
This constraint does force them to be `SpanExpectation` @zacharycmontoya 
Does this not accomplish what you're thinking about? https://github.com/DataDog/dd-trace-dotnet/pull/546/files/593302562e75915a279de2fd2720f2d285be3d95#r343942759

Reply:
You're right, it does accomplish what I'm thinking about. I guess I'm just being nitpicky in that I'd prefer strongly typing this to `SpanExpectation` than using generics and this constraint, but it's fine the way it is so I'm perfectly okay with this going in. Just expressing a coding preference 😄